lib/tests/streamlit/bootstrap_test.py

```diff
@@ -284,6 +284,33 @@ def test_print_old_git_warning(self, mock_git_repo):
             "Streamlit requires Git 2.7.0 or later, but you have 1.2.3." in out
         )
 
+    @patch("streamlit.config.get_config_options")
+    def test_load_config_options(self, patched_get_config_options):
```

Comment:
We also need to add some tests around config override scenarios. I don't want to bloat this PR, so please create a separate one for those. As always, happy to review them :)

Reply:
Ah, good point. I'll write down a note to get a test written for that once this last big push to get the rest of theming done is over